apps/aecore/test/aecore_forking_SUITE.erl

```diff
@@ -42,24 +42,27 @@ suite() ->
 init_per_suite(Config) ->
     ok = application:ensure_started(erlexec),
     DataDir = ?config(data_dir, Config),
-    TopDir = top_dir(DataDir),
-    Config1 = [{top_dir, TopDir}|Config],
-    make_shortcut(Config1),
+    TopDir = aecore_suite_utils:top_dir(DataDir),
+    Config1 = [{symlink_name, "latest.fork"},
+               {top_dir, TopDir},
+               {test_dir, "forking_SUITE"},
+               {config_name, "sync_SUITE"}] ++ Config,
```

Comment:
Why this config_name?

Reply:
so we could easily change it

Reply:
Why does a test_dir get the name of a SUITE as well as that a config has the nam of a SUITE? Hard to grasp what's going on here.

The test_dir is a temporary directory created for this SUITE.  atom_to_list(?MODULE) would do. Or rather the name of the Suite could be provided and later used to make a temp dir?? The key is that this is unique for each module, though. This can be enforced, rather then letting it be determined in the suite.

The config name clearly indicates that somehow that part of sync_SUITE should be isolated, because they base upon the same ground.

Reply:
The [config_name](https://github.com/aeternity/epoch/blob/master/apps/aecore/test/data/sync_SUITE.config) is the name of the config we start the virtual machine with. Currently both forking and sync suites use one file but this could possibly change.